app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -1 +1,31 @@
 /* Placeholder for backend dummy app */
+SpreePayuIntegration = {
```

Comment:
Sure, but how to reference this name from other parts of the system, when it is hidden in closure and we do not have any module system in place?

Please remember there is `SpreePayuIntegration.paymentMethodID` which needs to be set by external code in order for this javascripts to work properly.